dist/index.html

```diff
@@ -229,6 +230,47 @@ <h4 id='sleep-calendar-quality-average-weekly'></h4>
           </div>
         </section>
       </section>
+
+      <section class='card-container input-card-container' id='input-card-container'>
+        <section class='main-card' id='inputMainCard'>
+          <h3 class='input-card-title'>ADD A NEW LOG</h3>
+          <div class='input-card-top-row' id='inputCardTopRow'>
+            <!-- <button type='button' name='add-activity-button' class='add-activity-button'><i class="fas fa-shoe-prints"></i></button> -->
+            <i class="fas fa-shoe-prints" id='addNewActivity'></i>
+            <i class="fas fa-tint" id='addNewHydro'></i>
+            <i class="fas fa-bed" id='addNewSleep'></i>
+          </div>
+        </section>
+        <section class='input-card hide' id='addActivityCard'>
+          <h3>placeholder activity</h3>
+          <form>
```

Comment:
Great semantics, using a `<form>` element as a parent element to `<input>` elements.